api/ReadableStreamDefaultReader.json

```diff
@@ -262,14 +262,15 @@
         },
         "reject_pending_read_request": {
```

Comment:
Alternatively, should we just remove this entry and merge it into the `releaseLock()` entry? The new behavior is now well standardized and widely adopted.